AppDB/groomer.py

```diff
@@ -99,12 +100,16 @@ def __init__(self, zoo_keeper, table_name, ds_path):
     self.zoo_keeper = zoo_keeper
     self.table_name = table_name
     self.db_access = None
+    self.ds_access = None
```

Comment:
Yeah, when validating indices, there are some helper functions in datastore_server that are needed.